rtv/subreddit.py

```diff
@@ -73,6 +74,24 @@ def prompt_subreddit(self):
         if name is not None:
             self.refresh_content(order='ignore', name=name)
 
+    @SubredditController.register(Command('SUBREDDIT_FRONTPAGE'))
+    def show_frontpage(self):
```

Comment:
You should add a unit test for this, see **tests/test_subreddit.py** ``test_subreddit_prompt()`` for a similar example. There are instructions on how to run the tests in the bottom of the README, let me know if you need any help!